README.md

```diff
@@ -37,6 +37,11 @@ Speaking of contributing, read on!
 
 # Contributing
 
+|<img src="docs/static/meeting-icon.svg" alt="Developer Meetings" width="20" height="20" />Developer Meetings|
+|------------------|
+|We hold weekly developer meetings on a Thursday, to join them, watch previous meeting recordings or find more information, please see [the docs](https://docs.gomods.io/contributing/community/participating/). Absolutely everyone is invited to attend these, suggest topics, and participate!|
```

Comment:
Please use https://docs.gomods.io/contributing/community/developer-meetings/

Reply:
I copied and pasted the wrong tab haha. Nice spot!